smart-contracts-hardhat/scripts/deployUniversalProfiileWithLSP23.ts

```diff
@@ -0,0 +1,149 @@
+// libs
+import { ethers } from 'hardhat';
+import { AbiCoder } from 'ethers';
+import { ERC725 } from '@erc725/erc725.js';
+
+// LSPs artifacts
+import LSP23FactoryArtifact from '@lukso/lsp-smart-contracts/artifacts/LSP23LinkedContractsFactory.json';
+import UniversalProfileInitArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfileInit.json';
+
+// ERC725.js schemas
```

Comment:
Maybe we can specify this as well for the readers?

```suggestion
// ERC725.js Metadata schemas
```